recipes-nodes/rbuilder/rbuilder.inc

```diff
@@ -33,10 +34,11 @@ USERADD_PARAM:${PN} = "-r -s /bin/false -G eth rbuilder"
 do_install:append() {
     install -d ${D}${sysconfdir}/init.d
     install -m 0755 ${THISDIR}/init ${D}${sysconfdir}/init.d/rbuilder
-    
+
     # Install config files owned by root, readable by rbuilder
-    install -m 0744 ${THISDIR}/config ${D}${sysconfdir}/rbuilder.config
-    install -m 0744 ${THISDIR}/ofac.json ${D}${sysconfdir}/rbuilder.ofac.json
+    install -m 0644 ${THISDIR}/config.mustache ${D}${sysconfdir}/rbuilder.config.mustache
+    install -m 0640 ${THISDIR}/config.mustache ${D}${sysconfdir}/rbuilder.config
```

Comment:
pre-create the to-be rendered config to have the correct permissions. We don't want the file to be globally readable as it contains secrets.